homeassistant/components/sensor/iss.py

```diff
@@ -73,12 +79,28 @@ def name(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        return self._state
+        if self.iss_data:
+            return self.iss_data.is_above
+        else:
+            return STATE_UNKNOWN
 
     @property
     def device_state_attributes(self):
         """Return the state attributes."""
-        return self._attributes
+        if self.iss_data:
+            delta = self.iss_data.next_rise - datetime.utcnow()
+            attrs = {
+                ATTR_ISS_NUMBER_PEOPLE_SPACE:
+                    self.iss_data.number_of_people_in_space,
+                ATTR_ISS_NEXT_RISE: int(delta.total_seconds() / 60),
```

Comment:
We should store the next rise as an ISO8601 timestamp. All attributes should always be stored as absolute points (specific timestamp for example) instead of a relative attribute that would change whenever we call upadte.